pkgs/applications/misc/pdfstudio/pdfstudio.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
There is actually a function to generate such files (`makeDesktopItem`) on the fly. Please also remove the png logo. This should be directly available from unpacked debian package  (`opt/pdfstudio2021/.install4j/pdfstudio2021.png`).

Reply:
Thanks for pointing this out. Fixed.